README.md

```diff
@@ -103,7 +103,43 @@ Inside your `AppDelegate.application(_, handleEventsForBackgroundURLSession:comp
 
 ### Logging In
 
-Updated documentation coming soon!
+#### Invitation Code Authentication
+
+If a customer is being invited to a remote streem with an invitation code, authentication will happen using the following methods:
+
+```swift
+Streem.sharedInstance.login(withInvitationCode: code) { [weak self] error, details, identity in
```

Comment:
You need to also update the `Invitations` section later in this document with the correct invitation flow. Or maybe combine the two or something?